loadflow/loadflow-api/src/test/java/com/powsybl/loadflow/json/LoadFlowResultJsonTest.java

```diff
@@ -62,4 +67,17 @@ public void handleErrorTest() throws IOException {
         LoadFlowResultDeserializer.read(getClass().getResourceAsStream("/LoadFlowResultVersion10Error.json"));
     }
 
+    @Test
+    public void loadFlowResultJsonModuleTest() throws IOException {
+        ObjectMapper objectMapper = JsonUtil.createObjectMapper()
+                .registerModule(new LoadFlowResultJsonModule());
+        ObjectWriter writer = objectMapper.writerWithDefaultPrettyPrinter();
+        ByteArrayOutputStream os = new ByteArrayOutputStream();
+
+        writer.writeValue(os, createVersion11());
+        String result = new String(os.toByteArray(), StandardCharsets.UTF_8);
+
+        compareTxt(getClass().getResourceAsStream("/LoadFlowResultVersion11.json"), result);
```

Comment:
You should also test that you can deserialize the string properly through an ObjectReader.

Reply:
Agree: a good and complete test will do a complete round-trip. I think you should also update the `LoadFlowResultSerializer::write` and `LoadFlowResultDeserializer::read`. In these two methods we create manually the module: if you replace this part by the module you have just created you will cover your new code.